lib/Doctrine/ORM/Tools/Console/EntityManagerProvider/SingleManagerProvider.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Doctrine\ORM\Tools\Console\EntityManagerProvider;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\Tools\Console\EntityManagerProvider;
+
+class SingleManagerProvider implements EntityManagerProvider
+{
+    /** @var EntityManagerInterface */
+    private $entityManager;
+
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+    }
+
+    public function getManager(string $name = 'default'): EntityManagerInterface
+    {
+        if ($name !== 'default') {
+            throw UnknownManagerException::unknownManager($name, ['default']);
```

Comment:
Is it possible that people define several managers but end up using `SingleManagerProvider`? If yes, that exception will be confusing I think.

Reply:
Also, let's test this?

Reply:
No the `SingleManagerProvider` is for non Symfony context, where usually you have only one and you don't care about any of this. :)